cetmix_tower_server/models/cx_tower_variable_value.py

```diff
@@ -121,9 +121,11 @@ def _compute_value_char(self):
         for rec in self:
             if rec.variable_id.option_ids and rec.option_id:
                 rec.value_char = rec.option_id.name
+                rec.reference = rec.option_id.reference
```

Comment:
We should not replace variable value reference. It's doesn't depend on the option reference.